samples/RT-sample-app/nodejs/README.md

```diff
@@ -0,0 +1,149 @@
+
+## Run the app (Manually Uploading to Teams)
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+> the Teams service needs to call into the bot.
+
+### 1. Setup for Bot
+- In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2)
+
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+
+- While registering the bot, use `https://<your_tunnel_domain>/api/messages` as the messaging endpoint.
+    > NOTE: When you create your bot you will create an App ID and App password - make sure you keep these for later.
+
+### 2. Setup NGROK
+1) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http 3978 --host-header="localhost:3978"
+    ```
+   Alternatively, you can also use the `dev tunnels`. Please follow [Create and host a dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
+
+   ```bash
+   devtunnel host -p 3978 --allow-anonymous
+   ```
+
+### 3. Setup for code
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+2) In a terminal, navigate to `samples/RT-sample-app/nodejs`
+
+3) Install modules
+
+    ```bash
+    npm install
+    ```
+
+4) Update the `custom-environment-variables` configuration for the bot to use the `MicrosoftAppId` and `MicrosoftAppPassword`, `BaseUrl` with application base url.
+
+5) Update the `default` configuration for the bot to use the `appId` and `appPassword`.
+
+5) Run your app
+
+    ```bash
+    npm start
+    ```
+
+## RT Sample App Feature
+**This feature is to cover all 4 hero scenarios with one sample**
+1) Static tab that contains a dialog - 
+2) Static tab + ME => the static tab dependsOn ME
+3) Static tab + ME + BOT => the three mutually depends on each other
+4) actionME depends on Bot => for the bot-base-actionME scenario   
+
+## Running the sample
+
+**Install App:**
+
+![InstallApp](Images/1.Install_App.png)
+
+**Oneway Dependency and Mutual Dependency In Teams**
+
+![Teams](Images/2.PersonalBot.png)
+
+![Teams](Images/3.PersonalSelectMsgExtAction.png)
+
+![Teams](Images/4.PersonalMsgExtAction.png)
+
+![Teams](Images/5.PersonaMsgExtActionSubmit.png)
+
+![Teams](Images/6.StaticTab.png)
+
+![Teams](Images/7.TaskModule.png)
+
+**Outlook**
+
+**Outlook doesn't support bots, and because ComposeExtension depend on a bot, the ComposeExtension doesn't work in Outlook**
+![OutLookMessageExtension](Images/8.Outlook_SampleNotFoundForMsgExt.png)
+
+**Static Tab Depends On ComposeExtension, the Static tab doesn't work in outlook**
```

Comment:
Hmm, there're some nuances on Outlook's support for ME. Also composeExtension is already used in actionME's case as depending on bots, so we're creating a chain of dependency here, which makes sample app scenario too nuanced. 

The "staticTab dependsOn composeExtension" case actually applies better on Harmony. However, Harmony isn't ready for RT yet. 

So maybe let's do this - let's modify this scenario to be "Static Tab dependsOn bots" instead. Then just like the actionME scenario above, staticTab shouldn't show up in Outlook because Outlook doesn't support bots. Then the scenario is more straightforward and easy to understand. 

Sorry for the inconvenience.

Reply:
No Issues, will implement this scenario "**Static Tab dependsOn bots**"